yetanotherco/aligned_layer
# feat(explorer): Home page Displays Recent Batches with Information

# Home Page Displays Recent Batches with Information

## Description

The past home page displayed only the batch merkle root of a batch. This pr adds additional additional batch information: `Batch Hash`, `Status`, `Age`, and `Submission Block Number`. It also adds an optional header field for the table component to be used in a larger refactor.

<img width="889" alt="Screenshot 2024-12-18 at 04 45 47" src="https://github.com/user-attachments/assets/ca191b09-8cdc-4f5b-811c-559b57b8ee21" />

## To Test:
- Start a local devnet
- Run the explorer and observe the recent batches displayed on the homepage match the ones on the batcher page.

## Type of change

Please delete options that are not relevant.


## Checklist

  - [Link to the open issue addressing it]()
